modules/modules.nix

```diff
@@ -146,7 +146,7 @@ let
     (loadModule ./services/emacs.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/flameshot.nix { })
     (loadModule ./services/fluidsynth.nix { condition = hostPlatform.isLinux; })
-    (loadModule ./services/redshift-gammastep/gammastep.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./services/redshift-gammastep/gammastep.nix { })
```

Comment:
No systemd on darwin:
```suggestion
    (loadModule ./services/redshift-gammastep/gammastep.nix { condition = hostPlatform.isLinux; })
```

Reply:
This will break the tests.

Reply:
Should I change https://github.com/nix-community/home-manager/pull/1751/files#diff-dd7f089590e4d05731ef5488e4af6585a020228bb0b3cccbf18396c06c4b837aR77 to use a literal instead of link @rycee :thinking: ?

Reply:
Would be great to have an abstraction for launchd and systemd

Reply:
> No systemd on darwin:

BTW, we should add this to redshift too.

Reply:
Done in bd4562cceb3ef52e8d17ee96b84494eb514d072b, still needs some suggestion on how to fix manual in macOS.

Reply:
Can use `` `services.gammastep.settings` ``.